src/BurnableMintableCappedERC20.sol

```diff
@@ -45,6 +62,36 @@ contract BurnableMintableCappedERC20 is MintableCappedERC20 {
         _burn(account, balanceOf[account]);
     }
 
+    function burnFrom(address account, uint256 amount) public onlyOwner {
```

Comment:
I don't see testing for `burnFrom` and `permit`?

Reply:
I should push them today